src/main/java/com/microsoft/services/msa/LiveAuthClient.java

```diff
@@ -494,7 +495,22 @@ public void logout(LiveAuthListener listener) {
      * @param listener called on either completion or error during the logout process.
      */
     public void logout(Object userState, LiveAuthListener listener) {
-
+        this.logout(userState, listener, false);
+    }
+    
+    /**
+     * Logs out the given user.
+     *
+     * Also, this method clears the previously created {@link LiveConnectSession}.
+     * {@link LiveAuthListener#onAuthComplete(LiveStatus, LiveConnectSession, Object)} will be
+     * called on completion. Otherwise,
+     * {@link LiveAuthListener#onAuthError(LiveAuthException, Object)} will be called.
+     *
+     * @param userState arbitrary object that is used to determine the caller of the method.
+     * @param listener called on either completion or error during the logout process.
+     * @param dropCookies pass true to force the user to enter his credentials (again) on login
+     */
+    public void logout(Object userState, LiveAuthListener listener, boolean dropCookies) {
```

Comment:
Hmm, I think it would actually be appropriate to force a 'full logout' at every time.  Would you mind removing the boolean dropcookies and just clear out all cookies from this browser instance every time?  I see little utility in letting someone else choose to use the old misleading behavoir